std/math.d

```diff
@@ -5692,6 +5749,21 @@ private:
     }
 }
 
+///
+@safe unittest
+{
+    FloatingPointControl fpctrl;
+
+    fpctrl.rounding = FloatingPointControl.roundDown;
+    assert(lrint(1.5).feqrel(1.0) > 16);
```

Comment:
For demonstrating rounding we want exact equality, not `freqrel`. Same for the rest of this unittest.